package.json

```diff
@@ -110,7 +124,9 @@
     "start:lambda": "netlify-lambda serve src/lambda",
     "start:static": "gatsby build && gatsby serve",
     "serve": "gatsby serve",
-    "type-check": "tsc --noEmit"
+    "type-check": "tsc --noEmit",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook"
```

Comment:
Is the idea of this build command to have a new endpoint where we can access all the stories all the time for each new build?

Thinking about our ci/cd pipeline, where do you see this applied?

Reply:
I think he mostly addressed this here: https://github.com/ethereum/ethereum-org-website/pull/8497#issuecomment-1315825853